### PR TITLE
feat: browser start screenshot

### PR DESCRIPTION
**Pull request by nesquikm**
## Description

Add browser start screen screenshots for tabs.

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: lib/feature/browser/browser_tabs_view/browser_start_view.dart [hunk 1]
```diff
@@ -18,6 +23,9 @@ const _cardHeight = 116.0;
 /// Card viewport fraction
 const _cardViewportFraction = 0.85;
 
+/// How ofter we can take screenshot
```
Comment (Alex-A4) [inline lib/feature/browser/browser_tabs_view/browser_start_view.dart:26]: Misspell `often`

Comment (nesquikm) [reply lib/feature/browser/browser_tabs_view/browser_start_view.dart]: Fixed, thank you!

File: lib/feature/browser/browser_tabs_view/browser_start_view.dart [hunk 2]
```diff
@@ -64,14 +80,49 @@ class _BrowserStartViewState extends State<BrowserStartView> {
...
+        if (pngBytes == null) {
+          throw Exception("Can't convert image to bytes");
+        }
+
+        final file = File(BrowserTab.defaultImagePath);
+        await file.writeAsBytes(pngBytes);
+
+        await FileImage(File(BrowserTab.defaultImagePath)).evict();
+      } catch (e, s) {
+        _log.severe('Error while saving screenshot: $e', null, s);
```
Comment (Alex-A4) [inline lib/feature/browser/browser_tabs_view/browser_start_view.dart:121]: Why `$e` in 1-st param if second param of `severe` must contain error itself

Comment (nesquikm) [reply lib/feature/browser/browser_tabs_view/browser_start_view.dart:121]:
```dart
  void severe(Object? message, [Object? error, StackTrace? stackTrace]) =>
      log(Level.SEVERE, message, error, stackTrace);
```

So, I just put message and stack trace to the logs here.

**Review by Alex-A4 — APPROVED** [review]

**Pull request merged**